README.md

```diff
@@ -24,7 +24,31 @@ Please note that we plan for both implementations (gRPC C# native and fully-mana
 
 ## To start using gRPC for ASP.NET Core
 
-Documentation and guides are coming soon! In the mean time we suggest creating a basic website using the gRPC for ASP.NET template that comes with .NET Core 3.0 and refer to the examples at https://github.com/grpc/grpc-dotnet/tree/master/examples/Server.
+The best place to start using gRPC for ASP.NET Core is the gRPC template that comes with .NET Core 3.0. Use the template to [create a gRPC service website](https://docs.microsoft.com/en-us/aspnet/core/tutorials/grpc/grpc-start).
+
+Additional documentation and tutorials are available on docs.microsoft.com. Read more at [An introduction to gRPC on ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/grpc/).
+
+For additional examples of using gRPC in .NET refer to https://github.com/grpc/grpc-dotnet/tree/master/examples.
+
+## gRPC NuGet feed
+
+Official versions of gRPC are published to [NuGet.org](https://www.nuget.org/profiles/grpc-packages). This is the recommended place for most developers to get gRPC packages.
+
+Nightly versions of gRPC for ASP.NET Core are published to the gRPC NuGet repository at https://grpc.jfrog.io/grpc/api/nuget/v3/grpc-nuget-dev. It is recommended to use a nightly gRPC package if you are using a nightly version of .NET Core, and vice-versa. There may be incompatibilities between .NET Core and gRPC for ASP.NET Core if a newer version of one is used with an older version of the other.
+
+To use the gRPC NuGet repository and get the latest packages from it, place a `NuGet.config` file with the gRPC repository setup in your solution folder:
```

Comment:
It's probably easier to add a restore source instead.

Reply:
NuGet.config is the recommended way of doing it.